src/utils/logging.jl

```diff
@@ -241,10 +242,19 @@ MultiLogger([ConsoleLogger(stderr), SimpleLogger(stream)])
 ```
 """
 function MultiLogger(loggers::Array{T}) where {T <: Logging.AbstractLogger}
-    return MultiLogger(loggers, nothing)
+    return MultiLogger(loggers, nothing, Dict{Symbol, Base.LogLevel}())
```

Comment:
I chose `Symbol` instead of `String` because the default value for `group` is the filename containing the log message as a Symbol.